Makefile

```diff
@@ -42,6 +42,9 @@ push:
 
 olm:
 	oc create -f kubevirt-operator-configmap.yaml -f kubevirt-catalog-source.yaml
+	git clone -f https://github.com/kirankt/ember-csi-operator.git /tmp/ember-csi-operator
+	oc create -f /tmp/ember-csi-operator/deploy/olm-catalog/catalog-source.yml
```

Comment:
does this need to happen before the kubevirt catalogsource is created?  I guess order may not matter

Reply:
No it doesn't matter. 
The creation of the configmap and the catalog-source of the kubeVirt operator can be done **before or after** the creation of the configmap and the catalog-source of the ember operator.